src/wolframite/wolfram.clj

```diff
@@ -6320,29 +6320,28 @@
 (def ZipfDistribution "ZipfDistribution[ρ] represents a zeta distribution with parameter ρ.\nZipfDistribution[n, ρ] represents a Zipf distribution with range n." (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ZipfDistribution)))
 (def ZoomCenter "ZoomCenter is an option for DynamicImage that specifies the position of a zoom window within an image." (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ZoomCenter)))
 (def ZoomFactor "ZoomFactor is an option for DynamicImage that specifies the magnification factor of a zoom." (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ZoomFactor)))
-(def ..> "Maps to the Wolfram function RuleDelayed" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ..>)))
 (def ! "Maps to the Wolfram function Not" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote !)))
 (def = "Maps to the Wolfram function Set" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote =)))
 (def < "Maps to the Wolfram function Less" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote <)))
+(def =! "Maps to the Wolfram function Unset" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote =!)))
 (def || "Maps to the Wolfram function Or" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ||)))
-(def ..= "Maps to the Wolfram function SetDelayed" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ..=)))
 (def <= "Maps to the Wolfram function LessEqual" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote <=)))
 (def * "Maps to the Wolfram function Times" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote *)))
 (def -> "Maps to the Wolfram function Rule" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote ->)))
-(def fn "Maps to the Wolfram function Function" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote fn)))
 (def do "Maps to the Wolfram function CompoundExpression" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote do)))
 (def > "Maps to the Wolfram function Greater" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote >)))
+(def _> "Maps to the Wolfram function RuleDelayed" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote _>)))
+(def <_> "Maps to the Wolfram function Dot" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote <_>)))
```

Comment:
this should be `<*>`. How is this possible? Was this properly generated via `wolframite.impl.wolfram-syms.write-ns/write-ns!` ?!

Reply:
it was generated with write-ns, but it was a previous idea for the Dot symbol